client/src/pages/Report/NotificationSection/NotificationSection.tsx

```diff
@@ -55,8 +37,24 @@ const NotificationSection = () => {
         setShowModal(false);
     };
 
-    const handleDelete = (id: string) => {
-        setRows((prevRows) => prevRows.filter((row) => row.id !== id));
+    const handleDelete = (id: string, service: string) => {
+        setRows((prevRows) => {
+            // eslint-disable-next-line max-len
+            return prevRows.filter((row) => !(row.id === id && row.service === service));
+        });
+    };
+
+    const handleAddSelected = () => {
+        setRows((prevRows) => {
+            const newChannels = selectedChannels.filter(
+                (channel) => !prevRows.some(
+                    (row) => row.id === channel.id && row.service === channel.service
+                )
```

Comment:
Why do we need to check both id and service?

Reply:
We have 3 endpoints for notificatoin channels and entries from them might have same id, so we need to check both: id and service to avoid changing multiple entries